readme.md

```diff
@@ -263,7 +263,9 @@ remark()
   // Use `'error'` or `2` to treat messages as exceptions:
   .use(remarkLintFinalNewline, ['error'])
   .use(remarkLintFinalNewline, [2])
-  // Some rules accept options, and what they exactly accept is different for
+  // Some rules correspond to `remark-stringify` settings which they obey by default:
+  .use(remarkLintUnorderedListMarkerStyle).use({settings: {bullet: '*'}})
+  // Some accept options, and what they exactly accept is different for
```

Comment:
```suggestion
  // Some rules respond to shared settings that `remark-stringify` also understands:
  .use({settings: {bullet: '*'}})
  .use(remarkLintUnorderedListMarkerStyle)
  // Some rules accept options, and what they exactly accept is different for
```

(I do think that this new example should be *added*, after the existing examples, though)